apps/webservice/src/app/[workspaceSlug]/settings/(settings)/layout.tsx

```diff
@@ -3,5 +3,5 @@ export default function AccountSettingsLayout({
 }: {
   children: React.ReactNode;
 }) {
-  return <div className="m-10 mt-20">{children}</div>;
+  return <div className="mt-10">{children}</div>;
```

Comment:
This was a weird margin to have. It looks better this way on all the user setting pages.